app/models/installment.rb

```diff
@@ -21,11 +21,16 @@
 
 class Installment < ApplicationRecord
   include RankedModel
-  ranks :position
+  ranks :release_order
+  ranks :alternative_order
+
+  enum tag: {
+    side_story: 0,
+    spinoff: 1
```

Comment:
Assuming this is related to the abovementioned franchise tags, this will also need to be updated alongside that

Reply:
yes it is.